src/gulpfile.js

```diff
@@ -255,6 +255,29 @@ gulp.task('build-html', ['add-revision-numbers'], function () {
   .pipe(gulp.dest(paths.build));
 });
 
+gulp.task('build-mseditor', ['template-editor', 'add-revision-numbers'], function () {
+  var sources = gulp.src([
+    paths.build + '/template-editor/relayEditor.*.js'
+  ]);
+  return gulp.src([
+    paths.app + '/template-editor/index.html'
+  ])
+  .pipe(gulpInject(sources, {
+    addRootSlash: false, // ensures proper relative paths
+    ignorePath: paths.build + '/template-editor' // ensures proper relative paths
+  }))
+  .pipe(gulp.dest(paths.build + '/template-editor'));
```

Comment:
What do you think about keeping it in the same folder than the rest of the js files?

Reply:
You are talking about the bundle file or the index.html? If you refer to the bundle js file, yes, we can move it to the script folder in the build, I will do it.

Reply:
Oh, you are right, this line of code refers to the HTML.

The HTML is fine in the folder `template-editor`, I was talking about the .js file.